packages/dashboards/app/styles/navi-dashboards/components/dashboard-header.less

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright 2021, Yahoo Holdings Inc.
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ */
+
+.dashboard-header {
+  &__breadcrumb-link {
+    color: @navi-blue;
+  }
```

Comment:
Ideally we would use the `DenaliBreadcrumb` component, but for now we can just add the `link` class to the element and remove this style.